FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Eugene on 2021-09-09.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal enum NetworkErrorCode {
+	static let SUCCESS_200 = 200
+	static let REDIRECT_300 = 300
+	static let BAD_REQUEST_400 = 400
+	static let UNAUTHORIZED_401 = 401
+	static let FORBIDDEN_403 = 403
+	static let NOT_FOUND = 404
+	static let SERVER_500 = 500
+}
+
+internal class FeedImageMapper {
```

Comment:
The convention in this codebase is to omit the default access leve keyword of `internal`, please remove it.
```suggestion
class FeedImageMapper {
```